src/random_number.py

```diff
@@ -0,0 +1,4 @@
+from random import randint
+
+def get_random_number() -> int:
+    return randint(0, 2147483647)
```

Comment:
Potresti mettere `2147483647` in una costante

Reply:
potresti mettere o una variabile costante fuori dalla funzione, oppure lo metti come parametro in input, mettendo come valore di default del parametro 2147483647, tipo
```
def get_random_number(max_value = 2147483647) -> int:
```